.docker/Dockerfile

```diff
@@ -1,71 +1,47 @@
-FROM robotlocomotion/drake:1.29.0
-LABEL maintainer="Aditya Kamireddypalli kamireddypalliaditya@gmail.com"
+ARG ROS_DISTRO=rolling
+ARG RMW_IMPLEMENTATION=rmw_cyclonedds_cpp
+ARG DRAKE_VERSION=1.30.0-1
```

Comment:
I guess these are default values, maybe give it through docker compose?

Reply:
Overall done it with a couple of details: I couldn't build docker image without setting default argument inside dockerfile. So i just have updated docker-compose yaml instead of changing dockerfile as well.